MobileCenter/MobileCenter.xcodeproj/project.pbxproj

```diff
@@ -2058,13 +2058,15 @@
 					"DEBUG=1",
 					"$(inherited)",
 				);
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```

Comment:
Why do you need this in project.pbxproj? It is already in Global.xcconfig.

Reply:
When you built the SDK using Xcode 9 and then drag'n'drop the SDK in an Xcode 8 project, you end up with a linker error that says `ld: framework not found FileProvider for architecture x86_64` which I don't understand why it happens, but I have been able to repro this again and again.

Reply:
Then why does it only matter on MobileCenter project?